docset/windows/smbshare/set-smbshare.md

```diff
@@ -277,6 +277,28 @@ Default value: None
 Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
+### -LeasingMode
+Allow admins to stop leases & oplocks.
+
+- Full = default SMB3
+- Shared = grant read-caching lease but not write or handle-caching
+- None = no oplocks or leases, like modified SMB1/3
+
+> [!IMPORTANT]
+> Disabling Oplocks is not recommended by Microsoft and should never be used, but required by some older software, often due to using legacy database technology. Oplock disable is only a workaround - and your vendor should update to not require it. 
```

Comment:
```suggestion
> Disabling oplocks is not recommended by Microsoft and should never be done, but is required by some older software, often due to using legacy database technology. Disabling oplocks is only a workaround - and your vendor should provide an update so that it is not required. 
```

Reply:
I would use "...but may be required in some scenarios, often due to..." I would remove the term "older software" imho

Reply:
I'd go as far as "We do not recommend disabling oplocks." and leave it at that.

Reply:
Changes made as suggested, feel free to let me know of any changes needed.